tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -26,20 +31,16 @@ public void run() {
     @Override
     public void process(final Socket connection) {
         try (final var inputStream = connection.getInputStream();
-             final var outputStream = connection.getOutputStream()) {
-
-            final var responseBody = "Hello world!";
-
-            final var response = String.join("\r\n",
-                    "HTTP/1.1 200 OK ",
-                    "Content-Type: text/html;charset=utf-8 ",
-                    "Content-Length: " + responseBody.getBytes().length + " ",
-                    "",
-                    responseBody);
-
+             final var outputStream = connection.getOutputStream();
+             final BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream))
+        ) {
+            final String url = Objects.requireNonNull(bufferedReader.readLine());
+            final String requestUrl = url.split(" ")[REQUEST_URL_INDEX];
+            final ResponseMaker responseMaker = ResponseMakerFactory.findResponseMaker(requestUrl);
+            final String response = responseMaker.createResponse(requestUrl);
```

Comment:
```suggestion
    final String response = getResponse(requestUrl);

    private String getResponse(final String requestUrl)  {
        final ResponseMaker responseMaker = ResponseMakerFactory.findResponseMaker(requestUrl);
        final String response = responseMaker.createResponse(requestUrl);
        return response;
    }
```
메서드 분리해 보는 것은 어떨까요? :)